src/client/iomanager4file.cpp

```diff
@@ -111,6 +112,8 @@ void IOManager4File::UnInitialize() {
         scheduler_->Fini();
     }
 
+    discardTaskManager_.Stop();
```

Comment:
Can you point out which scenario can cause this problem? The `Stop` function first cancels all timed tasks according to its timerId, and in the last, it will wait until all tasks have been canceled or finished.